Forms

f-Xiaoxi opened the pull request:
-added fonts and colours to scss
-edited nav bar and made it sticky
-added header image with scroll spy(the link is not going to the exact position I'm hoping for though)
-edited the forms

will need to add diff nav bar to the header image

barbayjuliette commented:
- When clicking explore should go to the full page of index
-  Once logged in, should not go again to the header image but directly to the index
- Buttons: I think it would be nice if the color was a bit more transparent when you hover, and the text stays in a shade of yellow when hovering.
- Otherwise looks good

barbayjuliette commented on app/assets/stylesheets/components/_form_legend_clear.scss:
```diff
@@ -8,3 +8,212 @@
+  font-family: 'Lekton', sans-serif;
+}
+
+.meal-label-style:hover{
```
change meal

f-Xiaoxi replied:
removed

barbayjuliette commented on app/assets/stylesheets/application.scss:
```diff
@@ -19,6 +20,7 @@
 // Variables
 $green: #D0E3CC;
```
Can delete the green here that I had added

f-Xiaoxi replied:
Deleted

f-Xiaoxi commented:
When clicking explore should go to the full page of index
-Is not going to correct position. Will figure out later
Once logged in, should not go again to the header image but directly to the index
-fixed
Buttons: I think it would be nice if the color was a bit more transparent when you hover, and the text stays in a shade of yellow when hovering.
-fixed

barbayjuliette commented:
Can you also set the fonts / sizes to all the corresponding texts ? 
I mean for example in application.scss:p { font-family: $font-variable; ...}

f-Xiaoxi commented:
edited

barbayjuliette commented:
Oh okay, I see, I think there was a misunderstanding. When I said buttons, I meant all the buttons, the ones on the forms (log in, log out sign up). The one in the header image was perfectly fine as it was before

f-Xiaoxi commented:
ok, made the green login buttons slightly transparent when hover.
I think the header button looks better green when hovered so didn't make any changes to it.

barbayjuliette reviewed (approved):
Looks good, you can merge it.

f-Xiaoxi commented:
solved conflicts
